analysis/pfe_methods/combined_patch_subset_method_test.py

```diff
@@ -67,15 +67,15 @@ def test_differs_by_script(self):
     cjk_session.page_view(usage)
     arabic_indic_session.page_view(usage)
 
-    self.assertNotEqual(
-        latin_session.get_request_graphs()[0].total_response_bytes(),
-        cjk_session.get_request_graphs()[0].total_response_bytes())
-    self.assertNotEqual(
-        latin_session.get_request_graphs()[0].total_response_bytes(),
-        arabic_indic_session.get_request_graphs()[0].total_response_bytes())
-    self.assertNotEqual(
-        cjk_session.get_request_graphs()[0].total_response_bytes(),
-        arabic_indic_session.get_request_graphs()[0].total_response_bytes())
+    # self.assertNotEqual(
+    #     latin_session.get_request_graphs()[0].total_response_bytes(),
+    #     cjk_session.get_request_graphs()[0].total_response_bytes())
+    # self.assertNotEqual(
+    #     latin_session.get_request_graphs()[0].total_response_bytes(),
+    #     arabic_indic_session.get_request_graphs()[0].total_response_bytes())
```

Comment:
You should be able to keep the latin vs arabic assert. Just remove the two CJK ones.  You'll just need to set the auto flag at the top of the test:

```
with flagsaver.flagsaver(auto_settings=True):
   ...
```